app/application.py

```diff
@@ -126,3 +139,16 @@ def on_tray_activated(self, reason):
         if reason == QtWidgets.QSystemTrayIcon.DoubleClick:
             self.ui.show()
             self.ui.activateWindow()
+
+    def account_balance_zero(self):
+        if Profile.get_active().balance == 0:
+            # We only make a rpc-call, if we see in the profile, that the user has no coins
+            client = get_active_rpc_client()
+            if client.getbalance()['result'] == 0:
+                return True
+
+    def on_profile_changed(self, new_profile):
+        try:
+            self.ui.setDisabled(new_profile.balance == 0)
```

Comment:
If i spend all my coins the UI gets disabled without giving me any actionable information.